plant-and-cell-physiology.csl

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="6" et-al-use-first="6" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Plant and Cell Physiology</title>
+    <id>http://www.zotero.org/styles/plant-and-cell-physiology</id>
+    <link href="http://www.zotero.org/styles/plant-and-cell-physiology" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard-without-titles" rel="template"/>
+    <link href="http://www.oxfordjournals.org/our_journals/pcp/for_authors/general.html#advice" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="generic-base"/>
```

Comment:
@damnation333, we typically only use `field="generic-base"` for styles that can't be assigned to a single or small subset of disciplines, such as styles for general style guides (like APA) or for university. It should be deleted for almost all journals.